docs/api.md

```diff
@@ -1887,6 +1888,12 @@ The method evluates the XPath expression relative to the elementHandle. If there
 
 This method returns the bounding box of the element (relative to the main frame), or `null` if the element is not visible.
 
+#### elementHandle.boxModel()
+- returns: <[Promise]<?[Object]>>
```

Comment:
can you enumerate the Object fields here like we do in other fields? See `pptr.launch` for example